odoo/tools/misc.py

```diff
@@ -186,7 +186,7 @@ def file_path(file_path, filter_ext=('',), env=None):
     :raise ValueError: if the file doesn't have one of the supported extensions (`filter_ext`)
     """
     root_path = os.path.abspath(config['root_path'])
-    addons_paths = odoo.addons.__path__ + [root_path]
+    addons_paths = list(odoo.addons.__path__) + [root_path]
```

Comment:
Same thing in other places.

```suggestion
    addons_paths = [*odoo.addons.__path__, root_path]
```